tests/CMakeLists.txt

```diff
@@ -126,7 +126,7 @@ foreach(_var ${_variants})
     endif()
 endforeach(_var)
 
-set(_tests_rma local_rma simple_regular_exchange)
```

Comment:
why is this test gone?

Reply:
https://github.com/GridTools/GHEX/pull/48/commits/5b0986ac7a782f2de423da752c790aa8aa0cf36e

This is due to my incomplete cherry-pick. You have to re-add it after merging with benchmarks.